Make the HCMS Content Entry grey

ku5ic opened the pull request:
## Changes
This pull request introduces UI improvements and minor refactoring to Tabs components, along with a visual update to the FullScreenContentEntry container. The most significant changes are grouped below by theme.

**Tabs component styling:**

* Changed the default background of tab content to `wby-bg-neutral-base` and added margin and rounded corners for a more polished look. (`packages/admin-ui/src/Tabs/components/Content.tsx`)

**FullScreenContentEntry container update:**

* Updated the container background from `wby-bg-neutral-base` to `wby-bg-neutral-light` for a lighter appearance. (`packages/app-headless-cms/src/admin/views/contentEntries/ContentEntry/FullScreenContentEntry/FullScreenContentEntry.styled.tsx`)

<details>
<summary>SCREENSHOTS</summary>

<img width="1398" height="1351" alt="Screenshot 2025-10-23 at 12 02 35" src="https://github.com/user-attachments/assets/37d5f982-a2de-4f9d-93ba-7ebfa1dc5ed9" />


</details>

Closes [WEB-5373](https://webiny.atlassian.net/browse/WEB-5373)
## How Has This Been Tested?
n/a

## Documentation
n/a

ku5ic committed:
fix(admin-ui): update tab content and entry background colors

- Change tab content background from transparent to neutral-base
- Add margin and border radius to tab content styling
- Update full-screen content entry background to neutral-light

This improves visual hierarchy and consistency across the admin UI
components by applying appropriate background colors and spacing.

WEB-5373

adrians5j commented on packages/admin-ui/src/Tabs/components/Content.tsx:
```diff
@@ -4,7 +4,8 @@ import { cn, cva, type VariantProps } from "~/utils.js";
     [
-        "wby-bg-transparent",
+        "wby-bg-neutral-base",
+        "wby-my-lg wby-rounded-lg",
```
Not sure about `my-lg`. 

One thing: we now have more spacing here, in the dialog:

<img width="1003" height="791" alt="Image" src="https://github.com/user-attachments/assets/787a5993-8034-4440-bfe2-55272b76d101" />

This story:
```
http://localhost:6006/?path=/story/components-dialog--with-tabs
```

If I recall correctly, for tabs-in-dialog combination (which we discussed multiple times in the past), we decided to just add `bodyPadding: false`:
https://github.com/webiny/webiny-js/blob/8a0cadc5b69f045ce467316ac22a1223af5a49f9/packages/admin-ui/src/Dialog/components/DialogBody.tsx#L17

We can see it only removes x padding. Maybe it should also remove y padding as well?

IDK... 

What's also interesting is that... I was looking at this dialog here:

<img width="677" height="889" alt="Image" src="https://github.com/user-attachments/assets/647466ec-f714-446d-952e-b3f23bed1e1c" />

I was wondering.. how the hell did Pavel manage to do it without bodyPadding prop.

Then I realized this impl of his is wrong, because, in dialogs, you need to have a separator like this:

<img width="978" height="649" alt="Image" src="https://github.com/user-attachments/assets/cb231c2d-13cd-483a-9553-f1afeefa3e98" />

And if you were to use the prop (on the Tabs comp) that adds this separator, then it would not work, because the separator would not go all the way to the edges of the dialog. It would stop where x padding starts. 

What I'm trying to say is... with the change you did, and proper impl in this dialog, we'd have more y spacing then needed. 

And maybe this is the answer:
> We can see it only removes x padding. Maybe it should also remove y padding as well?

...but yeah, I'm not sure, we should test it a bit. Maybe add it and do some visual testing?

P.S. I'm not asking you to go into WB and fix the dialog with bodyPadding: false. Leave it if you want, it's not high prioirity. Looks fine. I might even tackle it during the tw4 upgrade.

adrians5j reviewed (changes requested):
It's complicated 😬

ku5ic committed:
fix(admin-ui): remove vertical padding when bodyPadding is false

Add !wby-py-none to DialogBody when bodyPadding prop is false to
ensure both horizontal and vertical padding are removed consistently.

adrians5j reviewed (approved):
Here we go!
